conf.d/supervisord.yaml.example

```diff
@@ -42,6 +42,8 @@ instances:
 #      - apache2       #           If not specified, the check will monitor all processes.
 #      - webapp
 #      - java
+#     tags:
+#      - processgroup:groupname
```

Comment:
Nit: I think there is only a one space diff between `tags` and `-processgroup:groupname`, we usually use 2, can you update ?